adapters/sovrn/sovrntest/supplemental/fpd.json

```diff
@@ -0,0 +1,168 @@
+{
+  "mockBidRequest": {
+    "id": "test-request-id",
+    "imp": [
+      {
+        "id": "test-imp-id",
+        "banner": {
+          "format": [
+            {
+              "w": 300,
+              "h": 250
+            },
+            {
+              "w": 300,
+              "h": 600
+            }
+          ]
+        },
+        "ext": {
+          "bidder": {
+            "tagid": "123456",
+            "anotherid": "654321"
+          }
+        }
+      }
+    ],
+    "device": {
+      "ua": "test-user-agent",
+      "ip": "123.123.123.123",
+      "language": "en",
+      "dnt": 0
+    },
+    "site": {
+      "domain": "www.publisher.com",
+      "page": "http://www.publisher.com/awesome/site",
+      "keywords": "word1",
+      "search": "search",
+      "ext": {
```

Comment:
According to requirements "data" in request will be a root element of (site/app/user). Then it will be merged to into (user/site/app).ext.data and sent to bidders. 
Expected request is correct. 

This feature is now in progress.  

Also are you going to support global and bidder-specific FPD permissions?